src/bitcoin-cli.cpp

```diff
@@ -104,14 +104,13 @@ static int AppInitRPC(int argc, char* argv[])
         return EXIT_FAILURE;
     }
     if (argc < 2 || HelpRequested(gArgs) || gArgs.IsArgSet("-version")) {
-        std::string strUsage = strprintf("%s RPC client version", PACKAGE_NAME) + " " + FormatFullVersion() + "\n";
+        std::string strUsage = PACKAGE_NAME "RPC client version " + FormatFullVersion() + "\n";
```

Comment:
Won't this cause the strings to be joined without any spacing?

Reply:
Fixed with force push...

Funnily, the missing space was even caught by CI: https://travis-ci.org/bitcoin/bitcoin/jobs/412055928#L3002